# avoid unnecessary linking with libm
# XML-LibXML uses none of its symbols
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -254,11 +254,6 @@ if ($config{LIBS} !~ /\-lz\b/) {
     }
     $config{LIBS} .= $is_Win32 ? ' -lzlib' :'';
 }
-
-if ($config{LIBS} !~ /\-lm\b/) {
-    # math support is important, but is not available separately in W32
-    $config{LIBS} .= $is_Win32 ? '' :' -lm';
-}
 # -------------------------------------------------------------------------- #
 
 # -------------------------------------------------------------------------- #
